### PR TITLE
LLM: improve transformers style API doc

### PR DESCRIPTION
**Pull request by plusbang**
## Description

Improvements on BigDL-LLM transformers-style API document page:
- change `llm.transformers.model` and `llm.transformers.modelling_bigdl` to `Hugging Face transformers AutoModel` and 
`Native Model`
- make classes more clear
- add some descriptions
- update autoclass path

Besides, fix the missing PyTorch API and LangChain API doc page.

### 4. How to test?
- [x] Document test (https://binbin-doc.readthedocs.io/en/improve-doc-v1/doc/PythonAPI/LLM/transformers.html)

### CODE REVIEW

File: docs/readthedocs/source/doc/PythonAPI/LLM/transformers.rst [hunk 1]
```diff
@@ -1,8 +1,14 @@
-BigDL-LLM `transformers`-style API
-=====================
+BigDL-LLM ``transformers``-style API
+====================================
 
-llm.transformers.model
----------------------------
+Hugging Face ``transformers`` Format
```
Comment (jason-dai) [inline docs/readthedocs/source/doc/PythonAPI/LLM/transformers.rst:4]: Hugging Face ``transformers`` AutoModel

File: docs/readthedocs/source/doc/PythonAPI/LLM/transformers.rst [hunk 2]
```diff
@@ -45,48 +57,59 @@ llm.transformers.model
 
 
 
-llm.transformers.modelling_bigdl
+Native Format
```
Comment (jason-dai) [inline docs/readthedocs/source/doc/PythonAPI/LLM/transformers.rst:60]: Native Model

File: docs/readthedocs/source/doc/PythonAPI/LLM/transformers.rst [hunk 3]
```diff
@@ -1,8 +1,14 @@
...
+Hugging Face ``transformers`` Format
+------------------------------------
+
+You can apply BigDL-LLM optimizations on any Hugging Face Transformers models by using the standard AutoModel APIs.
+
+
+AutoModelForCausalLM
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 .. autoclass:: bigdl.llm.transformers.model.AutoModelForCausalLM
```
Comment (Oscilloscope98) [inline docs/readthedocs/source/doc/PythonAPI/LLM/transformers.rst:13]: We need to show the import path as we suggested for users:

```rst
.. autoclass:: bigdl.llm.transformers.AutoModelForCausalLM
```

same for the other auto models.

File: docs/readthedocs/source/doc/PythonAPI/LLM/transformers.rst [hunk 4]
```diff
@@ -45,48 +57,59 @@ llm.transformers.model
...
 
-.. autoclass:: bigdl.llm.transformers.modelling_bigdl.ChatGLMForCausalLM
-    :members:
-    :undoc-members:
-    :show-inheritance:
-    :exclude-members: GGML_Model, GGML_Module, HF_Class
+    .. tab:: Llama
 
-    .. automethod:: from_pretrained
+        .. autoclass:: bigdl.llm.transformers.modelling_bigdl.LlamaForCausalLM
```
Comment (Oscilloscope98) [inline docs/readthedocs/source/doc/PythonAPI/LLM/transformers.rst:70]: Same here, we should use

```rst
.. autoclass:: bigdl.llm.transformers.LlamaForCausalLM
```

Same for the classes below.

**Comment by plusbang** [comment]
Have updated based on above comments.

**Review by Oscilloscope98 — APPROVED** [review]
LGTM

**Pull request merged**